### PR TITLE
Fix a quoting nit

### PR DESCRIPTION
**Pull request by pmorie**
:)

### CODE REVIEW

**Comment by duglin** [comment]
LGTM

File: pkg/controller/controller_binding.go [hunk 1]
```diff
@@ -600,7 +600,7 @@ func (c *controller) reconcileServiceBinding(binding *v1beta1.ServiceBinding) er
 
 		c.recorder.Event(binding, corev1.EventTypeNormal, successInjectedBindResultReason, successInjectedBindResultMessage)
 		glog.V(5).Infof(
-			`%s "%s/%s": Successfully bound to ServiceInstance ""%s/%s"" of ClusterServiceClass (K8S: %q ExternalName: %q) at ClusterServiceBroker %v`,
+			`%s "%s/%s": Successfully bound to ServiceInstance "%s/%s of ClusterServiceClass (K8S: %q ExternalName: %q) at ClusterServiceBroker %v`,
```
Comment (kibbles-n-bytes) [inline pkg/controller/controller_binding.go]: @pmorie It's missing the closing quotation mark. 😛 

`"%s/%s` -> `"%s/%s"`

**Comment by pmorie** [comment]
fffffffffff

**Comment by pmorie** [comment]
this PR is 33.33% error

**Pull request merged**